okio/src/main/kotlin/okio/platform.kt

```diff
@@ -37,9 +39,14 @@ expect annotation class JvmStatic()
 
 // TODO make internal https://youtrack.jetbrains.com/issue/KT-19664
 // TODO or remove after https://youtrack.jetbrains.com/issue/KT-18882
-@Target(FILE, FUNCTION)
+@Target(FILE, FUNCTION, PROPERTY_GETTER, PROPERTY_SETTER)
 expect annotation class JvmName(val name: String)
 
+// TODO make internal https://youtrack.jetbrains.com/issue/KT-19664
+// TODO or remove after https://youtrack.jetbrains.com/issue/KT-18882
+@Target(FIELD)
+expect annotation class Transient()
```

Comment:
Possible to define this in the JVM class only?

Reply:
aed0d0c